packages/design-system/src/components/N8nTooltip/Tooltip.vue

```diff
@@ -1,44 +1,33 @@
-<script lang="ts">
-import type { PropType } from 'vue';
-import { defineComponent } from 'vue';
+<script setup lang="ts">
 import { ElTooltip } from 'element-plus';
 import type { IN8nButton } from 'n8n-design-system/types';
 import N8nButton from '../N8nButton';
 
-export default defineComponent({
-	name: 'N8nTooltip',
-	components: {
-		ElTooltip,
-		N8nButton,
+export type Justify =
+	| 'flex-start'
+	| 'flex-end'
+	| 'start'
+	| 'end'
+	| 'left'
+	| 'right'
+	| 'center'
+	| 'space-between'
+	| 'space-around'
+	| 'space-evenly';
+
+const props = defineProps({
+	...ElTooltip.props,
+	content: {
+		type: String,
+		default: '',
+	},
+	justifyButtons: {
+		type: String as () => Justify,
```

Comment:
Use `PropType` for defining custom prop types:

```suggestion
		type: String as PropType<Justify>,
```